### PR TITLE
docs: Update graphql 

### PR DESCRIPTION
**Pull request by calldelegation**

### CODE REVIEW

**Review by matt-user — APPROVED** [review]

File: docs/reference/objects.mdx [hunk 1]
```diff
@@ -1667,6 +1701,20 @@ The status for a transaction that was squeezed out of the transaction pool.
...
 
+## `StateTransitionBytecode`
+
+The bytecode of a state transition upgrade.
+
+**fields:**
+
+`root`: [`[HexString!]`](/docs/reference/scalars/#hexstring)
+
+The merkle root of the new state.
```
Comment (netrome) [inline docs/reference/objects.mdx:1712]: @xgreenx correct me if I'm wrong, but this looks incorrect to me. This root should be the merkle root of the state transition bytecode, not the actual state.

Comment (xgreenx) [reply docs/reference/objects.mdx:1712]: Yes, it is merle root of the STF

File: docs/reference/queries.mdx [hunk 2]
```diff
@@ -217,6 +237,26 @@ Returns [`RelayedTransactionStatus`](/docs/reference/unions/#relayedtransactions
...
+## `stateTransitionBytecodeByVersion`
+
+Returns [`StateTransitionBytecode`](/docs/reference/objects/#statetransitionbytecode) details for a given version.
+
+**args:**
+
+`version`: `Int!`
+
+The version of the new state.
+
```
Comment (netrome) [inline docs/reference/queries.mdx:249]: Right same here: this is the version of the state transition function.

File: docs/reference/queries.mdx [hunk 3]
```diff
@@ -217,6 +237,26 @@ Returns [`RelayedTransactionStatus`](/docs/reference/unions/#relayedtransactions
...
+## `stateTransitionBytecodeByRoot`
+
+Returns [`StateTransitionBytecode`](/docs/reference/objects/#statetransitionbytecode) details for a given root.
+
+**args:**
+
+`root`: [`[HexString!]`](/docs/reference/scalars/#hexstring)
+
+The merkle root of the new state.
```
Comment (netrome) [inline docs/reference/queries.mdx:258]: ...of the state transition bytecode.

File: docs/reference/objects.mdx [hunk 4]
```diff
@@ -1897,6 +1957,22 @@ The maximum size allowed for a transaction.
...
+
+The bytecode that is uploaded.
+
+`uploadedSubsectionsNumber`: `Int`
+
+The number of uploaded subsections.
+
+`completed`: `Boolean!`
+
+Whether or not the bytecode has been uploaded.
```
Comment (netrome) [inline docs/reference/objects.mdx:1974]: I'd phrase this as "Whether or not the bytecode upload has completed." instead.

**Pull request merged**